vectordb/db/base.py

```diff
@@ -229,6 +229,14 @@ async def _deploy():
 
     def num_docs(self, **kwargs):
         return self._executor.num_docs()
+
+    def get_by_id(self,info_id, **kwargs):
+       ret=None
```

Comment:
can you please do better formatting, like `ret = None` is better than having no blank spaces.

Same applies in tests and in other lines.

Reply:
yeah it's done